src/App.js

```diff
@@ -1,16 +1,55 @@
 import React from 'react';
+import TodoForm from './components/TodoComponents/TodoForm';
+import TodoList from './components/TodoComponents/TodoList'
+
+const myData=[
+  {
+    task: 'Organize Garage',
+    id: 1528817077286,
+    completed: false
+  },
+  {
+    task: 'Bake Cookies',
+    id: 1528817084358,
+    completed: false
+  }
+];
 
 class App extends React.Component {
-  // you will need a place to store your state in this component.
-  // design `App` to be the parent component of your application.
-  // this component is going to take care of state, and any change handlers you need to work with your state
+  constructor() {
+    super();
+    this.state={
+      data:myData,
+       item:'',
+    };
```

Comment:
It seems like you are using the `item` state property as the `value` of your input field in the `TodoForm` component. If this is the case, it might be helpful to rename it (considering that the application is still small now) to something more declarative, like `todoInput` or `textInput`, for instance.